lib/zendesk_apps_tools/command.rb

```diff
@@ -63,6 +63,7 @@ def new
     desc 'validate', 'Validate your app'
     shared_options(except: [:unattended])
     def validate
+      require 'execjs'
```

Comment:
other than requiring it here, I think we should include it as a dependency here as well https://github.com/zendesk/zendesk_apps_tools/blob/master/zendesk_apps_tools.gemspec#L23

Reply:
💇‍♂️